packages/wonder-blocks-tabs/src/components/navigation-tabs.tsx

```diff
@@ -235,7 +235,7 @@ const styles = StyleSheet.create({
     },
     list: {
         // Add horizontal padding for focus outline of first/last elements
-        paddingInline: sizing.size_050,
+        paddingInline: sizing.size_040,
```

Comment:
The gap will also need to be updated to `sizing.size_160`! Noticed it by comparing the stories from the [snapshot diff](https://www.chromatic.com/test?appId=5e1bf4b385e3fb0020b7073c&id=67ed7611e053bbc80bf9f5a2)!

Reply:
Good call! I found a bunch of similar tokens. I was about to dig in to see what was causing the diffs, hopefully these fix them all!